pkg/util/fs/fs.go

```diff
@@ -190,25 +190,44 @@ func (h *fs) Copy(source string, dest string) (err error) {
 	return doCopy(h, source, dest)
 }
 
-func doCopy(h FileSystem, source, dest string) error {
-	// Our current symlink behaviour is broken.  We follow symlinks and copy the
-	// referenced file, and fail when a symlink is broken.  In all cases we
-	// should actually copy the symlink as is.  For now, at least catch the
-	// broken symlink case and copy the symlink rather than failing.
+func (h *fs) KeepSymlinks(k bool) {
+	h.keepSymlinks = k
+}
+
+func (h *fs) ShouldKeepSymlinks() bool {
```

Comment:
Needs godoc

Reply:
Thanks! Added godocs and also missed the bash completion